### DISCUSSION TITLE
Cannot Access EJBCA Admin WebUI

### DISCUSSION THREAD

**Discussion by halilimygel** [Keyfactor/ejbca-ce#326] [Q&A]
Hello,

Just a bit of background on the system, I am using:

- MariaDB 11.0.2
- WildFly 26
- EJBCA CE 7.11
- Apache Ant 1.9
- OpenJDK 11

I was able to deploy and install EJBCA following: https://doc.primekey.com/ejbca/ejbca-installation. But I am stuck on the last step of the installation process, "Finalizing the Installation". Specifically, it asks me to access the web UI, https://(serverIP):8443/ejbca/adminweb/. But I get an error saying it cannot provide a secure connection, which leads me to believe it's the certificate that I generated.
![image](https://github.com/Keyfactor/ejbca-ce/assets/140191950/7973c0f6-ad9b-4993-95e1-34f167a4b60f)

I got the superadmin.p12 certificate from _Ejbca root folder_/p12/superadmin.p12. I imported the certificate to my PC and I get the same error above. Is there anything I can do to debug this? I also attached my server.log and pastebin of the standalone.xml (Everything should be default in accordance with the EJBCA guide) of WildFly.

The only errors I could see in server.log were:

- ERROR [org.jboss.as.jsf] (MSC service thread 1-1) WFLYJSF0002: Could not load Jakarta Server Faces managed bean class: org.ejbca.ui.web.admin.peerconnector.PeerConnectorsMBean
- ERROR [org.cesecore.audit.AuditDevicesConfig] (ServerService Thread Pool -- 82) Duplicate property definitions of "securityeventsaudit.implementation.0". All defintions (2 occurrences) of this property will be ignored.

Another note, this is a reinstall of EJBCA. I rebooted the server on the first installation when I had this same problem and WildFly completely broke. 

Thanks.
[server.log](https://github.com/Keyfactor/ejbca-ce/files/12130594/server.log)
standalone.xml: https://pastebin.com/iXiwv55E

**Comment by primetomas** [comment]
The server.log only covers a small part of the installation. The log seems to stop even before "ant runinstall" finished. You should have run more commands after that, restarted wildfly etc?

I guess you modified some of the configuration files in conf/. Can you send those, for example install.properties.
